docroot/themes/humsci/humsci_basic/src/scss/utilities/_caption-credit.scss

```diff
@@ -1,6 +1,65 @@
 .field-media-image {
   + .field-media-image-caption {
     @include hb-caption-credit;
+    padding: hb-calculate-rems(16px) hb-calculate-rems(7px) hb-calculate-rems(8px) hb-calculate-rems(8px);
+    display: flex; //Flex is needed for image caption so the items floating right do not list in reverse order.
+    justify-content: flex-end;
+    flex-wrap: wrap;
+
+    @include grid-media-min('sm') { // the right side calculation is lower to adjust for the margin-right on the child elements
+      padding: hb-calculate-rems(24px) hb-calculate-rems(12px) hb-calculate-rems(16px) hb-calculate-rems(15px);
+    }
+
+    h1,
+    h2,
+    h3,
+    h4,
+    h5,
+    h6,
+    blockquote,
+    cite,
+    code,
+    li,
+    dd,
+    dt,
+    span,
+    strong,
+    em,
+    p,
+    .hs-button,
+    .hs-button--big,
+    .hs-secondary-button,
+    .hs-more-link,
+    .hs-external-link,
+    .hs-file-link,
+    .hs-pdf-link,
+    .hs-private-link {
+      display: inline-block;
+    }
+
+    code,
+    h1,
+    h2,
+    h3,
+    .hs-font-splash {
+      font-family: $hb-colorful-font--sans;
+    }
+
+    blockquote {
+      font-family: $hb-colorful-font--sans;
+
+      &:nth-child(n)::before,
+      &:nth-child(n)>p::before,
+      &:nth-child(n)>div::before,
+      &:nth-child(n)>p::after,
+      &:nth-child(n)>div::after {
```

Comment:
Let's add some spaces to make our tests happy. :smile:

![tests](https://user-images.githubusercontent.com/12678977/105870882-bb91f400-5fc6-11eb-902b-be2b0e92b8e9.png)

Reply:
I think we may want to add `&:nth-child(n)::after` as well.

You can also replace the `p` and `div` declarations with an `*` that would cover both instances.
So you could write, `&:nth-child(n) > *::before` and `&:nth-child(n) > *::after`. 
That one's your call. I'm cool with it it way. :tada:

Reply:
Yep, caught that in my new push, should already be fixed up with a new fetch. 😄